terraform/main.tf

```diff
@@ -8,15 +8,59 @@ terraform {
   }
 }
 
+data "aws_partition" "current" {}
```

Comment:
This should be placed (1) below the "base stuff" comment block, or (2) in a different file, explicitly calling out it's not part of the "minimal reference project".

Reply:
I saw the `Outdated` tag and thought I moved this 🤦‍♂ will fix

Reply:
This now lives under the `OPTION(custom_roles)` block!